devsetup/scripts/tripleo.sh

```diff
@@ -117,39 +172,92 @@ ip_address_suffix: ${IP_ADRESS_SUFFIX}
 interface_mtu: ${INTERFACE_MTU:-1500}
 ntp_server: ${NTP_SERVER}
 gateway_ip: ${GATEWAY}
+manage_default_route: ${TRIPLEO_NETWORKING}
 dns_server: ${PRIMARY_RESOLV_CONF_ENTRY}
-user_home: ${HOME}
+user_home: /home/zuul
```

Comment:
we should not hardcode the user here

Reply:
the issue on my setup was that on hypervisor I had no zuul user, but in VMs, it needed to be zuul. So that default HOME env var was incorrectly interpolated by j2 render.

To properly fix this we'd need to introduce a new env var for these scripts and Make file, somthing like CLOUD_USER, and get rid of all assumptions made here for zuul user, which goes a bit beyond of this patch scope